src/services/Peers.js

```diff
@@ -18,10 +19,39 @@ module.exports = {
     })
   },
 
+  formatUptimes: function(peers) {
+    function format(time) {
+      if (!time) return {name: '', time: -1};
+      var seconds = time;
+      var minutes = Math.floor(time / 60);
+      seconds -= minutes * 60;
+      var hours = Math.floor(minutes / 60);
+      minutes -= hours * 60;
+      var days = Math.floor(hours / 24);
+      hours -= days * 24;
+      var dayName = days + ' ' + 'Day' + (days > 1 ? 's' : '');
+      var hourName = hours + ' ' + 'Hour' + (hours > 1 ? 's' : '');
+      var minName = minutes + ' ' + 'min';
+      var secName = seconds + ' ' + 'sec';
+
+      if (days > 0) return { name: dayName, time: days * 60 * 60 * 24 };
+      else if (hours > 0) return { name: hourName, time: hours * 60 * 60 };
+      else if (minutes > 0) return { name: minName, time: minutes * 60 };
+      else return { name: secName, time: seconds };
+    }
+
+    _.map(peers, function(p) {
+      var f = format(p.uptime);
+      p.uptime_truncated = f.time;
```

Comment:
if we sort by uptime_truncated, than it'll insure no swaps in the region of the same uptime (say there are 20 peers with 25 Days)